### PR TITLE
add RealEstate Scrap

### PR DESCRIPTION
**Pull request by KPilosyan**

### CODE REVIEW

File: server/logs/log-11-11-2021.log [hunk 1]
```diff
@@ -0,0 +1,50 @@
+11-11-2021 16:30:05 [error]: ScraperController:start:FAILED {"ERROR":""}
```
Comment (hpilosyan) [inline server/logs/log-11-11-2021.log:1]: Do not commit your logs

File: server/scraper/ListAmScraper.js [hunk 2]
```diff
@@ -1,183 +1,151 @@
 import cheerio from 'cheerio';
```
Comment (hpilosyan) [inline server/scraper/ListAmScraper.js]: Rename this file to real estate scraper

File: server/services/VehicleEstimateService.js [hunk 3]
```diff
@@ -45,3 +45,17 @@ export default class VehicleEstimateService {
         }
     }
 }
+// const buildings = {
```
Comment (hpilosyan) [inline server/services/VehicleEstimateService.js]: What's this?

Comment (KPilosyan) [reply server/services/VehicleEstimateService.js]: Nothing. Forgot it was there.

**Pull request merged**